### PR TITLE
tile loop

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Fix way the parameters is specified

# Testing this PR
Check that https://ci.openmicroscopy.org/job/OMERO-DEV-merge-integration-java/ is still green

# Related reading

Found issue when working on https://github.com/openmicroscopy/openmicroscopy/pull/4823

cc @sbesson @chris-allan @joshmoore

### CODE REVIEW

**Comment by mtbc** [comment]
https://ci.openmicroscopy.org/job/OMERO-DEV-merge-integration-java/lastCompletedBuild/testngreports/integration/PyramidMinMaxTest/testRPSTileloop/ looks good.

**Comment by mtbc** [comment]
@rleigh-codelibre: I notice that `tiles.cpp` has `TileLoop::forEachTile` with `XYZTC`.

File: components/tools/OmeroJava/test/integration/PyramidMinMaxTest.java [hunk 1]
```diff
@@ -250,4 +251,35 @@ private Pixels importAndWaitForPyramid(File f, String format)
...
+     * @throws Exception
+     */
+    @Test
+    public void testRPSTileloop() throws Exception {
+        int sizeX = 256;
+        int sizeY = 256;
+        int sizeZ = 2;
+        int sizeT = 3;
+        int sizeC = 4;
+        Image image = mmFactory.createImage(sizeX, sizeY, sizeZ, sizeT,
```
Comment (mtbc) [inline components/tools/OmeroJava/test/integration/PyramidMinMaxTest.java]: argh, clearly our usual ordering is a bit inconsistent!

Comment (jburel) [inline components/tools/OmeroJava/test/integration/PyramidMinMaxTest.java]: yes!!!

**Comment by jburel** [comment]
I will check the ``Tiles.cpp``

**Comment by mtbc** [comment]
Good to merge.

**Comment by jburel** [comment]
No fix required in ``tiles.cpp`` unless we want to change the signature of the methods (not in this PR)

**Pull request merged**